packages/plugins/graphql/server/src/services/builders/filters/content-type.ts

```diff
@@ -85,11 +85,11 @@ export default ({ strapi }: Context) => {
 
         const isIDFilterEnabled = extension
           .shadowCRUD(contentType.uid)
-          .field('id')
+          .field('documentId')
```

Comment:
Don't we have constants for the document ID field name?

Reply:
We didn't use a const before. I'll refactor this later